.github/workflows/auto-start-ci.yml

```
@@ -59,11 +59,15 @@ jobs:
 
       - name: Setup @node-core/utils
         run: |
-          ncu-config set username ${{ secrets.JENKINS_USER }}
-          ncu-config set token "${{ secrets.GH_USER_TOKEN }}"
-          ncu-config set jenkins_token ${{ secrets.JENKINS_TOKEN }}
+          ncu-config set username $USERNAME
+          ncu-config set token $GH_TOKEN
+          ncu-config set jenkins_token $JENKINS_TOKEN
```

https://www.shellcheck.net/wiki/SC2086
```suggestion
          ncu-config set username "$USERNAME"
          ncu-config set token "$GH_TOKEN"
          ncu-config set jenkins_token "$JENKINS_TOKEN"
```